backend.js

```diff
@@ -108,45 +158,49 @@ Backend.prototype.getTile = function(z, x, y, callback) {
         }
 
         // Pass-thru of an upstream mapnik vector tile (not pbf) source.
-        if (data instanceof mapnik.VectorTile) return callback(null, data, headers);
+        if (data instanceof mapnik.VectorTile) return accept({data: data, headers: headers});
```

Comment:
Here the `callback( … )` call is replaced, but in line 157 above it's not. But a variable with the name `callback` doesn't exist any more in this scope.

Reply:
I removed it. The whole raw_buffer feature. It was added just recently ( although not reflected in the change log ) and in our stack we do not seem to use it anywhere.